### DISCUSSION TITLE
How to get unsorted output?

### DISCUSSION THREAD

**Discussion by seeCURE** [projectdiscovery/httpx#1111] [Q&A] [answered]
I am working with alexa top1m domains and I want the output of httpx to be not sorted. When I do something like `cat alexa-top1m.txt | httpx -mc 200` I get sorted output but I don't want that, I need the output according to the alexa-top1m.txt. Is there any option in `httpx` to achieve this?

Thanks in advance!

**Comment by ehsandeep** [comment, marked as answer]
@seeCURE below option should disable input sorting / dedup check.

```
   -sd, -skip-dedupe             disable dedupe input items (only used with stream mode)
```

You can try using `cat alexa-top1m.txt | httpx -mc 200 -s -sd`

**Reply by seeCURE** [reply]
Thank you very much for the answer @ehsandeep 🙏 and thank you again for what everything that you and your entire team at PD is doing for the community🙏 I really appreciate it!
